pkg/backends/jmeter/spec.go

```diff
@@ -24,5 +24,5 @@ func BuildLoadTestSpec(overwrite bool, distributedPods int32, testFileStr, testD
 	if testFileStr == "" {
 		return lt, ErrRequireTestFile
 	}
-	return loadTestV1.NewSpec(loadTestV1.LoadTestTypeJMeter, overwrite, distributedPods, testFileStr, testDataStr, envVarsStr, loadTestV1.ImageDetails{Image: masterImage, Tag: imageTag}, loadTestV1.ImageDetails{Image: workerImage, Tag: imageTag}, "", time.Duration(0)), nil
+	return loadTestV1.NewSpec(loadTestV1.LoadTestTypeJMeter, overwrite, distributedPods, testFileStr, testDataStr, envVarsStr, loadTestV1.ImageDetails{}, loadTestV1.ImageDetails{}, "", time.Duration(0)), nil
```

Comment:
`loadTestV1.ImageDetails` should not be empty, all fields must be applied when we create a new spec.

Reply:
This will be added almost immediately in the [next](https://github.com/hellofresh/kangal/blob/1788d2dceb8de77c9325b4f43f167bd74bcf8519/pkg/backends/jmeter/jmeter.go#L73) [sync](https://github.com/hellofresh/kangal/blob/1788d2dceb8de77c9325b4f43f167bd74bcf8519/pkg/controller/loadtest.go#L316)

Reply:
The proxy should give you valid LoadTest spec with all needed values, in controller previously it was a hack because setting specs didn't work properly. But right now it is not an issue anymore. In general, controller should not change resource specs - it should only change `Status` field.